_pages/install.md

```diff
@@ -13,37 +18,65 @@ link_actions:
 - '[![](/assets/install/android.svg) #Android](#android)'
 - '[![](/assets/install/docker.svg) #Docker](#docker)'
 - '[![](/assets/install/construction.svg) #Tools](#developer-tools)'
+- '[![](/assets/icons/build-circle.svg) Nightlies](/install/nightlies)'
+- '[![](/assets/icons/source-branch.svg) Source](/install/from_sources)'
```

Comment:
the assets are missing. for the nightly I'd rather use a 🌙 like icon

Reply:
I was trying to find a good icon for nightlies, but couldn't come up with anything great.
If you have a concrete icon in mind that fits into the material set, please let me know.

Reply:
Let's leave that one for now and fix it later.